### PR TITLE
include dnsimple api

### PR DESCRIPTION
**Pull request by pho3nixf1re**
Even though DNSimple is technically covered with lexicon not all systems can install python pip's easily. For these systems it is useful to have pure shell script API interactions.

### CODE REVIEW

File: dnsapi/dns_dnsimple.sh [hunk 1]
```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env sh
+
+# DNSimple domain api
+#
```
Comment (Neilpang) [inline dnsapi/dns_dnsimple.sh]: Please add your github repository link here.  so that one can report issue if there is any bugs.

**Comment by pho3nixf1re** [comment]
I ran out of time last weekend to implement that. I'll go ahead and do this over the coming weekend.

**Comment by Neilpang** [comment]
@pho3nixf1re 

How is going on?

**Comment by pho3nixf1re** [comment]
I'm pushing this up today, sorry for the delay.

**Comment by pho3nixf1re** [comment]
This should be ready for a final review with the removal feature implemented. I also resolved a conflict with the README file in the dnsapi folder.

**Comment by Neilpang** [comment]
@pho3nixf1re 
Please fix the format errors :  https://travis-ci.org/Neilpang/acme.sh/jobs/200967523

**Comment by pho3nixf1re** [comment]
Seems that my last change fixed it up.

**Comment by Neilpang** [comment]
Please do not change the function `_get()`.

If you want to use POST or PUT,  please use `_post()` function.

Please revert your changes in `acme.sh` file.

thanks.

**Comment by pho3nixf1re** [comment]
I made that change to do DELETE requests, not POST or PUT. What is your recommended way to create DELETE requests?

**Comment by Neilpang** [comment]
@pho3nixf1re 
you can use `_post()` function for any method, including DELETE

**Comment by Neilpang** [comment]
@pho3nixf1re 
How is going on ?

**Comment by pho3nixf1re** [comment]
I have not forgotten about this PR. Things got a little crazy over the last month (👶🍼). Expect an update in the next day or two.

**Comment by Neilpang** [comment]
No rush,  just take your time.  I just want to ping you.

**Comment by iler** [comment]
Any progress here?

**Comment by pho3nixf1re** [comment]
I'll have the requested changes up tonight. Sorry for all the delays. If it helps, it is working great for me as a local custom adapter as is.

**Comment by iler** [comment]
No worries :) I was just eager to actually get to test this!

**Comment by jasonkeller** [comment]
Any update on when this will be done?  Would be very useful in my lab environment.  Very excited to see this as I've recently switched to DNSimple.  Keep up the good work and know that it's much appreciated! :1st_place_medal:

**Comment by pho3nixf1re** [comment]
I reverted that change to the request functions and change the dnsimple request to use `_post` for delete requests. I've tested it out so this should be good to go now. So sorry for the wait on this!

**Pull request merged**